### DISCUSSION TITLE
ray()->queries() throwing BadMethodCallException

### DISCUSSION THREAD

**Discussion by wdavis** [spatie/ray#56] [Q&A] [answered]
`ray()->queries()` and `ray()->showQueries()` both throw `BadMethodCallException` exceptions for me.

Anyone else have this issue?

Ray 1.04
Laravel Ray 1.20

**Comment by freekmurze** [comment]
Could you create an issue for this. Please also include the backtrace.

**Comment by wdavis** [comment, marked as answer]
I had the wrong package installed. Should have been using `spatie/laravel-ray` instead of `spatie/ray`
